### ISSUE TITLE
LLM translations are not correctly attributed

### ISSUE THREAD

**Issue by flodolo** [mozilla/pontoon#3204]
Click the AI button and select rephrase, then click to copy the text and submit the translation.

The UX action log is correctly recorded (first `LLM dropdown select`, then `LLM dropdown copied`).

The translation is incorrectly attributed to Google Translate in the UI, and this code doesn't return any match

```
from pontoon.base.models import Translation

# List Translation copied from the LLM suggestions
Translation.objects.filter(machinery_sources__contains=['gpt-transform']).values("locale__code", "date")

>>> <TranslationQuerySet []>
```

**Comment by mathjazz** [comment]
Fixed in #3230.

**Comment by eemeli** [comment]
This isn't completely fixed yet, as the attribution is still wrong when the keyboard shortcut is used: https://github.com/mozilla/pontoon/blob/071829c8202dbb7ca713f4220370f8a125a11b4e/translate/src/modules/translationform/utils/editFieldShortcuts.ts#L112-L116
